docs/src/dataDisplay/popover/popoverSamplePlacement.jsx

```diff
@@ -1,58 +1,157 @@
-import React from 'react';
 import {
     Button,
-    Popover,
     Grid,
+    Popover,
+    Select,
 } from 'react-cm-ui';
+import {
+    upperCase,
+} from 'lodash';
+import React, {
+    useCallback,
+    useEffect,
+    useState,
+} from 'react';
 
-function PopoverSample() {
-    const popoverContent = (
-        <div>
-            Some cool tooltip content
-        </div>
-    );
-    return (
-        <Grid
-            spacing={2}
-            sm={4}
-        >
-            <Grid.Column>
-                <Popover
-                    content={popoverContent}
-                    placement="top"
-                >
-                    <Button>Top</Button>
-                </Popover>
-            </Grid.Column>
+function PopoverSamplePlacement() {
+    const [placementValue, setPlacementValue] = useState(null);
 
-            <Grid.Column>
-                <Popover
-                    content={popoverContent}
-                    placement="right"
-                >
-                    <Button>Right</Button>
-                </Popover>
-            </Grid.Column>
+    const placementOptions = [
+        {
+            label: 'bottom-end',
+            value: 'bottom-end',
+        },
+        {
+            label: 'bottom-start',
+            value: 'bottom-start',
+        },
+        {
+            label: 'bottom',
+            value: 'bottom',
+        },
+        {
+            label: 'left-end',
+            value: 'left-end',
+        },
+        {
+            label: 'left-start',
+            value: 'left-start',
+        },
+        {
+            label: 'left',
+            value: 'left',
+        },
+        {
+            label: 'right-end',
+            value: 'right-end',
+        },
+        {
+            label: 'right-start',
+            value: 'right-start',
+        },
+        {
+            label: 'right',
+            value: 'right',
+        },
+        {
+            label: 'top-end',
+            value: 'top-end',
+        },
+        {
+            label: 'top-start',
+            value: 'top-start',
+        },
+        {
+            label: 'top',
+            value: 'top',
+        },
+    ];
+
+    useEffect(() => {
+        setPlacementValue(placementOptions[2]);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
```

Comment:
Can we not list `placementOptions` as a dependency?
_Not a Blocker_

Reply:
No, because if we were this would be called on a loop. I just want it to be set once and left alone.

Reply:
Copy that.  It's hard to tell for me during a code review, so I'm always asking.
Resolving.